pkg/operator/configobservation/network/observe_network.go

```diff
@@ -101,3 +102,84 @@ func ObserveServicesSubnet(genericListers configobserver.Listers, recorder event
 
 	return out, errs
 }
+
+// ObserveExternalIPPolicy observes the network configuration and generates the
+// ExternalIPRanger admission controller accordingly.
+func ObserveExternalIPPolicy(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	listers := genericListers.(configobservation.Listers)
+
+	// set observed values
+	//  admission:
+	//    pluginConfig:
+	//      network.openshift.io/ExternalIPRanger:
+	//        configuration:
+	//          version: network.openshift.io/v1
+	//          kind: ExternalIPRangerAdmissionConfig
+	//          externalIPNetworkCIDRs: [...]
+	configPath := []string{"admission", "pluginConfig", "network.openshift.io/ExternalIPRanger", "configuration"}
+
+	// Need to handle 3 cases:
+	// 1. retrieval error: pass through existing, if it exists
+	// 2. Null egress policy: disable the admission controller by returning no config
+	// 3. Egress policy: enable the egress controller, pass through configuration.
```

Comment:
"egress" in the comments is wrong. ("**e**xternalip" + "in**gress**" = "egress"?)

Reply:
hah. fixed.